smartapps/smartthings/lifx-connect.src/lifx-connect.groovy

```diff
@@ -32,11 +33,16 @@ mappings {
 	path("/test") { action: [ GET: "oauthSuccess" ] }
 }
 
-def getServerUrl() { return "https://graph.api.smartthings.com" }
-def apiURL(path = '/') { return "https://api.lifx.com/v1beta1${path}" }
+def getServerUrl()               { return "https://graph.api.smartthings.com" }
+def getShardUrl()                { return getApiServerUrl() }
+def getCallbackUrl()             { return "https://graph.api.smartthings.com/oauth/callback"}
+def apiURL(path = '/') 			 { return "https://api.lifx.com/v1${path}" }
 def buildRedirectUrl(page) {
```

Comment:
Upon further review this method doesn't even look like its used so all the more reason to delete :)